packages/ui-react/src/modules/container.ts

```diff
@@ -0,0 +1,7 @@
+import 'reflect-metadata'; // include once in the app for inversify (see: https://github.com/inversify/InversifyJS/blob/master/README.md#-installation)
```

Comment:
Perhaps this is not necessary, given that it's done from `common` as well?

Reply:
Considering it's a singleton and we import the package from `common`, I think we can skip this. The Inversify docs caution against importing the package multiple times, as `Reflect` should be a global singleton.

Reply:
Thanks Rachid!